e2e/scenario_test.go

```diff
@@ -600,33 +549,6 @@ func Test_ubuntu1804gpu(t *testing.T) {
 	})
 }
 
-func Test_ubuntu1804gpu_azurecni(t *testing.T) {
```

Comment:
are we loosing the only 1804gpu test we had ?

Reply:
we had two of them